src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/ResearchSetup.java

```diff
@@ -288,6 +288,7 @@ public static void setupResearches() {
         register("portable_teleporter", 278, "Teleportation from Anywhere", 42, SlimefunItems.PORTABLE_TELEPORTER);
         register("trident", 279, "Trident", 20, "TRIDENT");
         register("farmer_talisman", 280, "Talisman of the Farmer", 18, SlimefunItems.TALISMAN_FARMER);
+        register("rainbow_armor", 281, "Wear the Rainbow", 22, SlimefunItems.RAINBOW_HELMET, SlimefunItems.RAINBOW_CHESTPLATE, SlimefunItems.RAINBOW_LEGGINGS, SlimefunItems.RAINBOW_BOOTS);
```

Comment:
Not sure I like the research name? This one wont block anything however. What about `Show your true colors` ? If not ignore this comment.

Reply:
Gonna give it a little of my own flavor since im changing up so much anyways